src/doc_builder/build_doc.py

```diff
@@ -213,7 +221,7 @@ def build_mdx_files(package, doc_folder, output_dir, page_info):
             "The deployment of the documentation will fail because of the following errors:\n" + "\n".join(all_errors)
         )
 
-    return anchor_mapping
+    return anchor_mapping, src_py_path_mapping
```

Comment:
`src_py_path_mapping` is used to know wich `mdx` file to updated when a docstring is updated inside `python` file